docs/installation.md

```diff
@@ -161,3 +161,10 @@ The usual multiqc command line should work fine:
 ```
 docker run -v "$PWD":"$PWD" -w "$PWD" ewels/multiqc multiqc .
 ```
+# Using MultiQC through Galaxy
```

Comment:
Please add a header level to all headers here. eg. `##`for this one and `###` for the other two.

Please also add a newline or two before this line.